### DISCUSSION TITLE
(Base) how to get an item not by key but by data inside

### DISCUSSION THREAD

**Discussion by SuperErnD** [deta/cloud-docs#324] [Help] [answered]
im on python

**Comment by jnsougata** [comment, marked as answer]
Make some query to the base!

https://docs.deta.sh/docs/base/sdk/#fetch

**Reply by SuperErnD** [reply]
> Сделайте запрос к базе!
> 
> https://docs.deta.sh/docs/base/sdk/#fetch

thanks

**Reply by SuperErnD** [reply]
> Make some query to the base!
> 
> https://docs.deta.sh/docs/base/sdk/#fetch

not working i using code 
```py
user = app.db['users'].fetch({
        "email": form.email,
        "password": form.password
    })
```
and returns 
```json
{
  "_count": 0,
  "_last": null,
  "_items": []
}
```
i correct creating query?

**Reply by SuperErnD** [reply]
i fixed, thanks
